### PR TITLE
Fix code in docs/questions about MCP

### PR DESCRIPTION
**Pull request by wjayesh**
## Describe changes
I implemented a fix the for the code in our docs and have some questions about the MCP.

## Pre-requisites
Please ensure you have done the following:
- [ ] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have based my new branch on `develop` and the open PR is targeting `develop`. If your branch wasn't based on develop read [Contribution guide on rebasing branch to develop](https://github.com/zenml-io/zenml/blob/main/CONTRIBUTING.md#-pull-requests-rebase-your-branch-on-develop).
- [ ] If my changes require changes to the dashboard, these changes are communicated/requested.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: docs/book/user-guide/advanced-guide/data-management/model-management.md [hunk 1]
```diff
@@ -204,6 +209,7 @@ particular model version. Possible options for stages are:
 - `latest`: The latest version of the model. This is a virtual stage to retrieve the latest version only - versions cannot be promoted to `latest`.
 - `archived`: This is archived and no longer relevant. This stage occurs when a
   model moves out of any other stage.
+  > is this archived or "No stage"?
```
Comment (avishniakov) [inline docs/book/user-guide/advanced-guide/data-management/model-management.md:212]: It is Archived, we never fall back to no stage.

File: docs/book/user-guide/advanced-guide/data-management/model-management.md [hunk 2]
```diff
@@ -117,9 +117,14 @@ dashboard and API functionality to support the full ML lifecycle.
 
 Model versions are created implicitly as you are running your machine learning
 training, so you don't have to immediately think about this. If you want more
-control over how and when these versions are controlled, our API has you
+control over how and when these versions are controlled, 
+
+> our API has you
```
Comment (avishniakov) [inline docs/book/user-guide/advanced-guide/data-management/model-management.md:122]: `our API has you covered` - it continues. Is it bad English or? 🙂

File: docs/book/user-guide/advanced-guide/data-management/model-management.md [hunk 3]
```diff
@@ -117,9 +117,14 @@ dashboard and API functionality to support the full ML lifecycle.
...
 covered, with options for whether new model versions are created as well as for
-the deletion of new model versions when pipeline runs fail. [See
+the deletion of new model versions when pipeline runs fail.
+
+is this still true? can't find these options in ModelVersion anymore.
```
Comment (avishniakov) [inline docs/book/user-guide/advanced-guide/data-management/model-management.md:126]: Nah, good catch. I would rephrase the whole paragraph into something like
```
Model versions are created implicitly as you are running your machine learning
training, so you don't have to immediately think about this. If you want more
control over versions, our API has you covered, with an option to explicitly
name your versions.
```

**Review by avishniakov — CHANGES_REQUESTED** [review]

File: docs/book/user-guide/advanced-guide/data-management/model-management.md (hunk 3 above)
Comment (wjayesh) [reply docs/book/user-guide/advanced-guide/data-management/model-management.md:126]: got it!

File: docs/book/user-guide/advanced-guide/data-management/model-management.md (hunk 1 above)
Comment (wjayesh) [reply docs/book/user-guide/advanced-guide/data-management/model-management.md:212]: okay! i just noticed "no stage" on the dashboard for older versions, so I asked.

**Review by avishniakov — APPROVED** [review]
